### ISSUE TITLE
`Global CSS cannot be imported from within node_modules` error with Next.js

### ISSUE THREAD

**Issue by fnick851** [pmndrs/leva#90]
```sh
./node_modules/react-colorful/dist/index.css
Global CSS cannot be imported from within node_modules.
Read more: https://err.sh/next.js/css-npm
Location: node_modules/leva/dist/leva.cjs.dev.js
```

Next.js gives this error when trying to use Leva. 
Here's Next.js' doc page regarding this issue: https://github.com/vercel/next.js/blob/master/errors/css-npm.md

I wonder if Leva would consider follow its recommendation. 
Or is there a workaround for this? 

Thanks.

**Comment by dbismut** [comment]
Hi! The package is still under heavy development and we don't have a release strategy in place yet. 

Bu the good news is that this is already fixed on the master branch!

**Comment by dbismut** [comment]
Should be fixed in 0.3.x, let me know if otherwise!

**Comment by fnick851** [comment]
@dbismut 
It works now. Thanks for making this great widget.
